### PR TITLE
Revamped cloudnative.nz website

### PR DESCRIPTION
**Pull request by iiamabby**
## What has been changed? 
Changed the flow of the original website, restructured and organised the content. 
Added sunburst chart for career opportunities in cloud computing.
Created a web scraper to pull in the good first issues from the cloudnative.nz repo.
Made content more accessible for screen readers and zoomable text. 
Tooltips to breakdown the jargon into easier to understand, bite size pieces.
Added a community page to highlight some of the mahi we have been involved with recently. 
Minor style changes.
Created a contribution guide as the main README @Riaankl can you read through this please and amend as needed? 

## What needs doing? 
We need to create some issues so the web scraper can do its thing.
TGC has agreed to create some art work for the banners for each page.
The blog section needs building out still, this will be one of the issues.
Linkage between ii.nz and cloudnative.nz

### CODE REVIEW

File: about/index.html
Comment (riaankleinhans) [inline about/index.html:1]: please change cpy right date to 2023

File: README.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+
+Take some time to understand the project's purpose, goals, and existing codebase. Read the project's documentation and get familiar with its structure.
+
+### Step 3: Set Up Your Development Environment
+
+Follow the project's instructions to set up your development environment. This may involve installing Git, Node.js, and other dependencies.
+
+### Step 4: Create a GitHub Account
+
+Most open source projects use GitHub for collaboration. Create a GitHub account if you don't have one already.
```
Comment (riaankleinhans) [inline README.md]: Many open-source projects use the collaborative power of GitHub. If you want to contribute to these projects you need to set up a GitHub account at www.github.com. Choose your username carefully, as that would be your "band" in the open-source community.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+
+## Contributing to cloudnative.nz 
+
+### Step 5: Downloading the source code
+For this site follow these steps to start contributing:
+
+- Create a github account 
+- Fork this repo
```
Comment (riaankleinhans) [inline README.md]: [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repo

File: README.md [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+- Create a github account 
+- Fork this repo 
+- Run `git clone https://github.com/<yourusername>/site.git` in your command line to pull down the forked repo to your device 
+- Run this website by opening the index.html file from your file explorer
+- View the issues and decide on which one to work through, ask questions or ask for assistance in the thread of the issue if you need to.
+
+
+### Step 6: Submit a Pull Request
+
+When your changes are ready, submit a pull request to the original project repository. A pull request is a request to merge your changes into the main codebase. Be sure to follow the project's contribution guidelines and provide a clear description of your changes.
```
Comment (riaankleinhans) [inline README.md:45]: Do we have a "contribution guide" ? If yes, let's hyperlink it.
If not, please make open an issue. You can assign me to it. I can help with that

File: README.md [hunk 4]
```diff
@@ -1,3 +1,69 @@
...
+
+### Step 7: Engage with the Community
+
+Engage with the community by participating in discussions, commenting on issues, and helping other contributors. Collaboration is a key aspect of open source, and being an active community member enhances your open source experience.
+
+## Adding New Content
+
+### Step 9: Create New Pages or Sections
+
+Do you have ideas for new sections or pages for the website? Whether it's a new tutorial, a glossary, or an FAQ section, you can propose these additions to the project maintainers. Open an issue on the repository to discuss your ideas and get feedback from the community.
```
Comment (riaankleinhans) [inline README.md:55]: Open an [issue](https://github.com/cloudnative-nz/site/issues) in the repository to discuss .....

File: README.md [hunk 5]
```diff
@@ -1,3 +1,69 @@
...
+
+### Step 10: Implement Your Changes
+
+If your ideas are well-received, implement the changes by adding new pages or sections to the website. Follow the project's code and style guidelines to ensure consistency.
```
Comment (riaankleinhans) [inline README.md:59]: Maybe we can add a "code and style" guideline to the contribution guide?

File: README.md [hunk 6]
```diff
@@ -1,3 +1,69 @@
...
+
+### Step 11: Submit Another Pull Request
+
+Once your changes are ready, submit another pull request with the new content. Remember to document your additions and explain their purpose in the pull request description.
+
```
Comment (riaankleinhans) [inline README.md:64]: Remeber to [link you issue to the pull requiest](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to help everyone keep track of the work.

File: README.md (hunk 3 above)
Comment (riaankleinhans) [reply README.md:45]: The "Blog Contribute.md do not quite tick this box? WDYT?

File: blog/contribute.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+Clone the forked repository to your local machine using Git. Open your terminal and run the following command:
+
+Replace `yourusername` with your GitHub username and `yourproject` with the name of the repository.
+
+### Step 3: Create a New Branch
+
+Create a new branch for your changes. Use a descriptive name for the branch that reflects the purpose of your blog post.
+
+
```
Comment (riaankleinhans) [inline blog/contribute.md]: Delete extra lines

File: blog/contribute.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+### Step 6: Customize Your Page Design
+
+You have the freedom to design your blog page to your liking. You can use HTML, CSS, and JavaScript to add styling, layout, and interactive elements. However, please ensure that the navbar and footer remain consistent with the rest of the website.
+
+### Step 7: Commit and Push Your Changes
+
+Once you've added your blog post and customized your page design, it's time to commit your changes and push them to your forked repository.
+
+
+
```
Comment (riaankleinhans) [inline blog/contribute.md]: Delete extra lines

File: README.md [hunk 9]
```diff
@@ -1,3 +1,69 @@
...
+### Step 5: Downloading the source code
+For this site follow these steps to start contributing:
+
+- Create a github account 
+- Fork this repo 
+- Run `git clone https://github.com/<yourusername>/site.git` in your command line to pull down the forked repo to your device 
+- Run this website by opening the index.html file from your file explorer
+- View the issues and decide on which one to work through, ask questions or ask for assistance in the thread of the issue if you need to.
+
```
Comment (riaankleinhans) [inline README.md:41]: Delete extra line

File: community/index.html [hunk 10]
```diff
@@ -0,0 +1,459 @@
...
+                        Tauranga<br />
+                        <a href="tel:0220648277">022 064 8277</a>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </footer>
+    <div class="copyright">
+        <div class="container">
+            &copy; Copyright 2022 HE WAKA EKE NOA. All Rights Reserved.
```
Comment (riaankleinhans) [inline community/index.html:349]: Copyright 2023

File: community/index.html [hunk 11]
```diff
@@ -0,0 +1,459 @@
...
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                    <div class="row mt-15 mb-15">
+                        <div class="col-md-6">
+                            <h2>Connect with us <span>Whakapā Mai</span></h2>
+
+                            <div class="inline-row">
+                                <div class="col-3"><i class="fa-solid fa-2x fa-location-dot"></i></div><div class="col-9">
+                                    <p class="lg">
+                                        Unit 3, 63 Farm Street<br />
```
Comment (riaankleinhans) [inline community/index.html]: Let's update this to either the iimaginerium address or maybe just a email address?

File: opportunities/index.html [hunk 12]
```diff
@@ -165,25 +155,32 @@ <h2 class="styled">Training &amp; Development <span></span></h2>
...
                             </div>            </div>
                     </div>
                 </div>
             </section>
-
+            <section>
+                <div class="container">
+                    <h2 class="styled">Good First Issues <span></span></h2>
+                    <p>A good first issue is a label that can be set on Issues, usually these are easy to navigate, have heaps of instruction and direction on how to implement the needed changes</p>
+                    <p>Get involved with open source today by contributing to this website</p>
```
Comment (riaankleinhans) [inline opportunities/index.html:180]: Hyperlink to contributing instructions

**Review by riaankleinhans — COMMENTED** [review]
a few improvements suggestions. Will review the website again once we got it published.

File: README.md (hunk 4 above)
Comment (iiamabby) [reply README.md:55]: Done #11

File: opportunities/index.html (hunk 12 above)
Comment (iiamabby) [reply opportunities/index.html:180]: Will do this when we have a contribution guide

File: README.md (hunk 3 above)
Comment (iiamabby) [reply README.md:45]: I created an issue #12 for the contribution guide, I agree with the "Blog Contribute.md" I will remove this for now as is mentioned in #11, will create a seperate issue for the styling guide

File: README.md (hunk 1 above)
Comment (iiamabby) [reply README.md]: Changed, thanks @Riaankl

File: README.md (hunk 2 above)
Comment (iiamabby) [reply README.md]: Done

File: blog/contribute.md (hunk 8 above)
Comment (iiamabby) [reply blog/contribute.md]: Done

File: community/index.html (hunk 10 above)
Comment (iiamabby) [reply community/index.html:349]: Done

File: community/index.html (hunk 11 above)
Comment (iiamabby) [reply community/index.html]: Removed address,  set as Tauranga, New Zealand as discussed

**Pull request merged**